From 48ff651c521a07e50870f39a9ab5f4cc2ad76d03 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Mon, 11 Dec 2017 19:41:12 +0100
Subject: [PATCH] apparmor: copy xpdfimport definition into xpdfimport and
 allow "m"

Dec 11 18:46:11 sid kernel: [  176.085106] audit: type=1400 audit(1513014371.237:22): apparmor="DENIED" operation="file_mmap" profile="libreoffice-xpdfimport" name="/usr/lib/libreoffice/program/xpdfimport" pid=1511 comm="xpdfimport" requested_mask="m" denied_mask="m" fsuid=1000 ouid=0

Change-Id: I31ddb412d1dd95506a00d22739a33d2e6770ba23
---
 sysui/desktop/apparmor/program.xpdfimport | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/sysui/desktop/apparmor/program.xpdfimport b/sysui/desktop/apparmor/program.xpdfimport
index 3934c0a04006..aa76e85505fe 100644
--- a/sysui/desktop/apparmor/program.xpdfimport
+++ b/sysui/desktop/apparmor/program.xpdfimport
@@ -22,6 +22,8 @@ profile libreoffice-xpdfimport INSTDIR-program/xpdfimport {
   /usr/share/libreoffice/share/config/* r,
   owner @{HOME}/.config/libreoffice{,dev}/?/user/uno_packages/cache/log.txt rw,
 
+  INSTDIR-program/xpdfimport        pxm,
+
   #Uncomment for build testing (should be one directory <- of instdir)
   #/mnt/store/git/libo/**    r,
 }
-- 
2.11.0

